### PR TITLE
Make every packages incompatible with conduit 3.0.0 (+ fix conf-llvm on fedora)

### PR DESCRIPTION
**Pull request by kit-ty-kate**
cc @mseri @dinosaure

### CODE REVIEW

File: packages/git/git.1.4.10/opam [hunk 1]
```diff
@@ -48,7 +48,7 @@ depopts: [
 conflicts: [
   "cohttp" {<= "0.15.0"}
   "cohttp" {>= "0.18.0"}
-  "conduit" {< "0.6.0"}
+  "conduit" {< "0.6.0" | >= "3.0.0"}
```
Comment (mseri) [inline packages/git/git.1.4.10/opam:51]:
```suggestion
  "conduit" {< "0.6.0" }
  "conduit" {>= "3.0.0"}
```

File: packages/git/git.1.4.11/opam [hunk 2]
```diff
@@ -41,7 +41,7 @@ depopts: [
 conflicts: [
  "cohttp"  {<= "0.15.0"}
  "cohttp"  {>= "0.18.0"}
- "conduit" {< "0.6.0"}
+ "conduit" {< "0.6.0" | >= "3.0.0"}
```
Comment (mseri) [inline packages/git/git.1.4.11/opam:44]:
```suggestion
  "conduit" {< "0.6.0" }
  "conduit" {>= "3.0.0"}
```

File: packages/git/git.1.4.4/opam [hunk 3]
```diff
@@ -40,7 +40,7 @@ depopts: [
 conflicts: [
   "cohttp" {<= "0.15.0"}
   "cohttp" {>= "0.18.0"}
-  "conduit" {< "0.6.0"}
+  "conduit" {< "0.6.0" | >= "3.0.0"}
```
Comment (mseri) [inline packages/git/git.1.4.4/opam:43]:
```suggestion
  "conduit" {< "0.6.0" }
  "conduit" {>= "3.0.0"}
```

File: packages/git/git.1.4.5/opam [hunk 4]
```diff
@@ -40,7 +40,7 @@ depopts: [
 conflicts: [
   "cohttp" {<= "0.15.0"}
   "cohttp" {>= "0.18.0"}
-  "conduit" {< "0.6.0"}
+  "conduit" {< "0.6.0" | >= "3.0.0"}
```
Comment (mseri) [inline packages/git/git.1.4.5/opam:43]:
```suggestion
  "conduit" {< "0.6.0" }
  "conduit" {>= "3.0.0"}
```

File: packages/git/git.1.4.6/opam [hunk 5]
```diff
@@ -40,7 +40,7 @@ depopts: [
 conflicts: [
   "cohttp" {<= "0.15.0"}
   "cohttp" {>= "0.18.0"}
-  "conduit" {< "0.6.0"}
+  "conduit" {< "0.6.0" | >= "3.0.0"}
```
Comment (mseri) [inline packages/git/git.1.4.6/opam:43]:
```suggestion
  "conduit" {< "0.6.0" }
  "conduit" {>= "3.0.0"}
```

File: packages/git/git.1.4.7/opam [hunk 6]
```diff
@@ -40,7 +40,7 @@ depopts: [
 conflicts: [
   "cohttp" {<= "0.15.0"}
   "cohttp" {>= "0.18.0"}
-  "conduit" {< "0.6.0"}
+  "conduit" {< "0.6.0" | >= "3.0.0"}
```
Comment (mseri) [inline packages/git/git.1.4.7/opam:43]:
```suggestion
  "conduit" {< "0.6.0" }
  "conduit" {>= "3.0.0"}
```

File: packages/git/git.1.4.8/opam [hunk 7]
```diff
@@ -40,7 +40,7 @@ depopts: [
 conflicts: [
   "cohttp" {<= "0.15.0"}
   "cohttp" {>= "0.18.0"}
-  "conduit" {< "0.6.0"}
+  "conduit" {< "0.6.0" | >= "3.0.0"}
```
Comment (mseri) [inline packages/git/git.1.4.8/opam:43]:
```suggestion
  "conduit" {< "0.6.0" }
  "conduit" {>= "3.0.0"}
```

File: packages/git/git.1.4.9/opam [hunk 8]
```diff
@@ -40,7 +40,7 @@ depopts: [
 conflicts: [
   "cohttp" {<= "0.15.0"}
   "cohttp"  {>= "0.18.0"}
-  "conduit" {< "0.6.0"}
+  "conduit" {< "0.6.0" | >= "3.0.0"}
```
Comment (mseri) [inline packages/git/git.1.4.9/opam:43]:
```suggestion
  "conduit" {< "0.6.0" }
  "conduit" {>= "3.0.0"}
```

File: packages/git/git.1.5.0/opam [hunk 9]
```diff
@@ -45,7 +45,7 @@ depopts: [
 conflicts: [
  "cohttp"  {< "0.18.0"}
  "cohttp"  {>= "0.19.0"}
- "conduit" {< "0.8.4"}
+ "conduit" {< "0.8.4" | >= "3.0.0"}
```
Comment (mseri) [inline packages/git/git.1.5.0/opam:48]:
```suggestion
  "conduit" {< "0.8.4" }
  "conduit" {>= "3.0.0"}
```

File: packages/git/git.1.5.1/opam [hunk 10]
```diff
@@ -49,7 +49,7 @@ depopts: [
 conflicts: [
  "cohttp"  {< "0.18.0"}
  "cohttp"  {>= "0.19.0"}
- "conduit" {< "0.8.4"}
+ "conduit" {< "0.8.4" | >= "3.0.0"}
```
Comment (mseri) [inline packages/git/git.1.5.1/opam:52]:
```suggestion
  "conduit" {< "0.8.4" }
  "conduit" {>= "3.0.0"}
```

File: packages/git/git.1.5.2/opam [hunk 11]
```diff
@@ -38,7 +38,7 @@ depends: [
   "mirage-http" {with-test}
   "mirage-fs-unix" {with-test & = "1.1.4"}
   "cohttp" {with-test}
-  "conduit" {with-test}
+  "conduit" {with-test & < "3.0.0"}
```
Comment (mseri) [inline packages/git/git.1.5.2/opam:41]:
```suggestion
  "conduit" {< "0.8.4" }
  "conduit" {>= "3.0.0"}
```

File: packages/git/git.1.5.2/opam [hunk 12]
```diff
@@ -55,7 +55,7 @@ depopts: [
 conflicts: [
  "cohttp"  {< "0.18.0"}
  "cohttp"  {>= "0.19.0"}
- "conduit" {< "0.8.4"}
+ "conduit" {< "0.8.4" | >= "3.0.0"}
```
Comment (mseri) [inline packages/git/git.1.5.2/opam:58]:
```suggestion
  "conduit" {< "0.8.4" }
  "conduit" {>= "3.0.0"}
```

File: packages/git/git.1.5.3/opam [hunk 13]
```diff
@@ -57,7 +57,7 @@ depopts: [
 conflicts: [
  "cohttp"  {< "0.18.0"}
  "cohttp"  {>= "0.19.0"}
- "conduit" {< "0.8.4"}
+ "conduit" {< "0.8.4" | >= "3.0.0"}
```
Comment (mseri) [inline packages/git/git.1.5.3/opam:60]:
```suggestion
  "conduit" {< "0.8.4" }
  "conduit" {>= "3.0.0"}
```

Comment (mseri) [reply packages/git/git.1.5.3/opam:60]: There are a lot of these for the various git versions. I stop marking them, we can fix them using a sed call all at once

File: packages/irmin/irmin.0.9.10/opam [hunk 14]
```diff
@@ -56,6 +56,7 @@ depopts: [
 conflicts: [
   "cohttp" {< "0.18.3"}
   "git"    {< "1.7.0" | >= "1.8.0"}
```
Comment (mseri) [inline packages/irmin/irmin.0.9.10/opam:58]: Independent of this PR: Shouldn’t this be split on two lines as well?

**Comment by mseri** [comment]
If I recall correctly we needed to split conflicts on multiple lines otherwise the constraints were not correctly interpreted. In this case we need to fix all of the conflicts sections

**Comment by kit-ty-kate** [comment]
mh, I don't recall something like that. Are you referring to conflicts with `{.. & ..}` in them instead? I used `opam admin add-conflicts` for this.

**Comment by mseri** [comment]
Maybe it is that, I could not find an issue or a PR where I had seen it. That is why I was asking. So it is an issue only with &? Then we can merge this

**Comment by kit-ty-kate** [comment]
Yeah it should be fine. There are many other packages with this pattern in opam-repository already as well. If this is really an issue we can change it later but it should be equivalent. Merging then.

File: packages/git/git.1.5.0/opam [hunk 15]
```diff
@@ -45,7 +45,7 @@ depopts: [
 conflicts: [
  "cohttp"  {< "0.18.0"}
  "cohttp"  {>= "0.19.0"}
```
Comment (mseri) [inline packages/git/git.1.5.0/opam:47]: I still cannot find the issue, I only found https://github.com/ocaml/opam-repository/issues/15441 which is somewhat unrelated. But look at the two lines above (and the same kind of repeated constraints are in many other packages)

Comment (mseri) [reply packages/git/git.1.5.0/opam:47]: Tested locally and the `|` works fine

**Pull request merged**